smoke_test/steps/visitors_page.rb

```diff
@@ -0,0 +1,29 @@
+module SmokeTest
+  module Steps
+    class VisitorsPage < BaseStep
+      PAGE_PATH = '/request'
+
+      def validate!
+        if page.current_path != PAGE_PATH
+          fail "expected #{PAGE_PATH}, got #{page.current_path}"
+        end
+      end
+
+      # rubocop:disable Metrics/AbcSize
```

Comment:
As with my other comment, is it worth re-enabling this after the method it affects? (Though I realise there's not much after it just yet…)